### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by Munyabelden**
-Added index.html filled in with code 
-Added style.css and added styles to the header
-Created text.md and add it to the gitignore file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,24 @@
...
-After you're finished please remove all the comments and instructions!
--->
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+<!-- PROJECT DESCRIPTION -->
```
Comment (huseyinbiyik) [inline README.md]: [REQUIRED]As the previous reviewer requested [here](https://github.com/Munyabelden/helloSetup/pull/1#discussion_r1068291892), please remove the placeholder data and fill in the project information. React.js, express.js they are just there as placeholders.

File: README.md [hunk 2]
```diff
@@ -1,76 +1,24 @@
...
 
-</div>
+## Hello Microverse
```
Comment (huseyinbiyik) [inline README.md:3]: [REQUIRED]As the previous reviewer requested [here](https://github.com/Munyabelden/helloSetup/pull/1#discussion_r1068293800), please add customized information on your Getting Started section headings.


[OPTIONAL] Please customized all the sections, replace them with your project and personal data. Please remove all the instructions targeting you. For example: "Describe how a new developer could make use of your project."

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @Munyabelden,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] Your commit messages are good but the first word of your commit message needs to be capitalized. 📝
- [OPTIONAL] Your text.md file needs to be ignored before pushing to Github. So, you need to remove it manually from Github.com
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Munyabelden** [comment]
Hi I opened a new pull request without comments and then I saw after that I
shouldn't do that qnd my reviews are finished

I don't know what to do

here is the link to the new pull request:
https://github.com/Munyabelden/helloSetup/pull/3

**Pull request closed without merging**